package.json

```diff
@@ -29,7 +29,8 @@
   "scripts": {
     "start": "yarn build && yarn serve",
     "build": "yarn build-scss && yarn build-js && yarn build-class-references",
-    "build-scss": "sass --load-path=node_modules --embed-sources --style=compressed scss:build/css && postcss --use autoprefixer --replace 'build/css/**/*.css' --map",
+    "build-tokens": "((style-dictionary build --verbose --config ./sd.config.json) || true)",
```

Comment:
Why all the parentisis and `|| true`? Not sure what's meant to be doing. Looks quite different to standard command acll.

Reply:
See PR description - the `|| true` is intentional (and temporary), so that errors thrown by `style-dictionary` builds due to not having the style dictionary files will not cause `build-scss` to fail.

In between this PR being merged and style dictionary being merged, style dictionary will not have a complete configuration so its builds will fail. This `||true` gives us compatibility so that PRs not related to style dictionary do not fail to build in the meantime. It can be removed once style dictionary is merged